### ISSUE TITLE
add templating system

### ISSUE THREAD

**Issue by echolabstech** [coderbunker/coderbunker.com#56]
### What problem are we solving ###
We need an easy way to create new content on the website. Currently, we build pages from scratch by copy pasta (copy and paste). Templating will allow us to build pages faster using components an imports.

### todo (if appropriate):
- [x] install Jekyll
- [x] move nav into a component
- [x] move footer into a component
- [x] move header into a component
- [x] rebuild index.html with components
- [x] document process of how to build the app with Jekyll

_____________________________________________________________________________________________________________________
### reviewer todo (if appropriate):
- [ ] at least one reviewer has seen this PR work on local
- [ ] this PR does not break tests

**Comment by echolabstech** [comment]
we decided to migrate one page to jekylls and let interns and juniors do the other pages so we can test documentation and use this as a training opportunity.

**Comment by lautaro450** [comment]
I installed jekyll in my local environment (ubuntu desktop 18.04)
Documentation to install jekyll:
* [Requirements](https://jekyllrb.com/docs/installation/)
* [Installation in Ubuntu Linux](https://jekyllrb.com/docs/installation/ubuntu/) state: **Tested**
* [Installation in Windows](https://jekyllrb.com/docs/installation/windows/) state: **Tested**
* [Installation in Mac OS](https://jekyllrb.com/docs/installation/macos/) state: **We need to test it**

**GitHub Pages is currently using Jekyll 3.8.5. We need to install the same version to avoid compatibility issues.**

* [x] nav component inside _includes/menu.html
* [x] footer component inside _includes/footer.html
* [x] header component inside _includes/header.html
* [x] landingPage layout inside _layouts/landingPage.html
* [x] workshops landingPage inside landingPages/workshops.html state: **not finished**

I need to keep working on the following:

* [ ] rebuild index.html
* [ ] document the process

**Comment by lautaro450** [comment]
@echolabstech  Check landingPages/workshops.html. I was testing how the variables in jekyll works. I added variables like "descriptionBigImage", "subject", "title", "subTitle" and then I used this variables inside the template. I think it would be possible to optimize this in order to create an easy way to add content for non-engineer people.
![coderbunker com jekyll](https://user-images.githubusercontent.com/15842049/56756170-beddc780-67c3-11e9-88a4-5c154b59371d.png)
